src/manifold/src/quickhull.cpp

```diff
@@ -85,34 +228,94 @@ void QuickHull::buildMesh(const VertexDataSource& pointCloud, bool CCW,
   // The planar case happens when all the points appear to lie on a two
   // dimensional subspace of R^3.
   planar = false;
-  createConvexHalfEdgeMesh();
+  createConvexHalfedgeMesh();
   if (planar) {
-    const size_t extraPointIndex = planarPointCloudTemp.size() - 1;
-    for (auto& he : mesh.halfEdges) {
-      if (he.endVertex == extraPointIndex) {
-        he.endVertex = 0;
+    const int extraPointIndex = planarPointCloudTemp.size() - 1;
+    for (auto& he : mesh.halfedges) {
+      if (he.endVert == extraPointIndex) {
+        he.endVert = 0;
       }
     }
     originalVertexData = pointCloud;
     planarPointCloudTemp.clear();
   }
-}
 
-ConvexHull QuickHull::getConvexHull(const VertexDataSource& pointCloud,
-                                    bool CCW, bool useOriginalIndices,
-                                    double epsilon) {
-  buildMesh(pointCloud, CCW, useOriginalIndices, epsilon);
-  return ConvexHull(mesh, originalVertexData, CCW, useOriginalIndices);
+  // reorder halfedges
+  Vec<Halfedge> halfedges(mesh.halfedges.size());
+  Vec<int> counts(mesh.halfedges.size(), 0);
+  Vec<int> mapping(mesh.halfedges.size());
+  Vec<int> faceMap(mesh.faces.size());
+
+  // reorder halfedges
+  // Since all faces are not used now (so we should start from 0 and just
```

Comment:
This is a bit confusing to read. I think this means 'some faces are disabled and should not go into the halfedge vector'.